### PR TITLE
Feat/create home page

### LINKED ISSUE #2: User Story 1:  As a user when I visit the app I should be greeted with an intro home page and nav bar.

### LINKED ISSUE #3: User Story 2: As a user when click the Exhibits button I should be taken to see all the different exhibits.

### LINKED ISSUE #10: User Story 5: As a user when I click the home button no matter where I am, I will be taken to the home page.

### PR DESCRIPTION
**Pull request by LeftyLincoln**
## Description

This request will build out the components, run our fetch for a randomDog, and then connect our Genre to run a fetch for that specific breed of dog. This request will also implement Router functionality to create dynamic URL's for when certain actions are taken by the user. 

## Related Issues

Closes #2 
Closes #3 
Closes #10 

## Changes

Main change was adding Router with specific routes.

### CODE REVIEW

**Comment by Lexyful** [comment]
## Review
Great job team! The Page is looking good and all the dogs are being displayed properly in each exhibit!

## What's next
Now we can get the Context API implemented and see if that can minimize the prop drilling.

File: src/Components/App/App.js [hunk 1]
```diff
@@ -4,27 +4,51 @@ import { fetchData } from '../../data/apiCall';
...
-      <DogContext.Provider value={{ text, setText }}>
+      <DogContext.Provider>
         <Nav />
-        <Home />
+        <Switch>
+          <Route path="/dogexhibits/:breed" render={({match} ) => <BreedPage breed={match.params.breed} breedUrls={breedUrls}/> } />
+          <Route path="/dogexhibits" render={() => <ExhibitContainer selectBreed={selectBreed}/> } />
+          <Route exact path="/" render={() => <Home />} />
+          <Redirect from="*" to="/"/> 
+        </Switch>
```
Comment (hfaerber) [inline src/Components/App/App.js:51]: Good use of switch!

File: src/Components/App/App.js [hunk 2]
```diff
@@ -4,27 +4,51 @@ import { fetchData } from '../../data/apiCall';
...
   const [error, setError] = useState('')
 
   const fetchDogData = () => {
-   fetchData('breed/schnauzer/images')
-    .then(data => console.log(data))
+    console.log('fetch dog')
+  
+    console.log('hi',breed)
+    fetchData(`breed/${breed}/images`)
+    .then(data => {
```
Comment (hfaerber) [inline src/Components/App/App.js:22]: I love being able to see your workflow here of testing it out with a hardcoded path then making it dynamic.  Nice work!

**Review by hfaerber — COMMENTED** [review]
Nice work!  I've added some comments for your consideration :)

**Pull request merged**